auth/index.js

```diff
@@ -0,0 +1,5 @@
+const {router} = require('./router');
+require('./strategies');
```

Comment:
Any reason we can't export a function from strategies that we call in `index.js`?